frontend/components/AwardPdf.js

```diff
@@ -43,6 +102,7 @@ const createDoc = (props) => {
             <Text style={styles.header}>Congratulations, {recipientName}!</Text>
             <Text style={styles.message}>You have been awarded the {awardTypeName} award by {creatorName}.</Text>
             <Text style={styles.date}>{moment(granted).format('lll')}</Text>
+            <Image style={styles.signature} src={ url: valid-url, method: 'GET', headers: {}, body: '' }/>
```

Comment:
i think the `src` prop here should be 
```
src={{ url: props.signatureUrl, ... }} 
```
with the two curlys because its an object within a prop.

also, might fail if the prop isn't passed (not sure how react-pdf handles it). Might be a good idea for the interim while we implement on both the server and the client to do something like 
```
...
{
    props.signatureUrl ? (
        <Image style={styles.signature} src={{ url: props.signatureUrl, method: 'GET', headers: {}, body: ''} }/>
    ) : null
}
...
```

Reply:
Good call!